src/components/PricingCard/PricingCardAttributeItem.react.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+
+import Icon from "../Icon";
+// TODO: Add RootComponent prop
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +included?: boolean,
+  // FIXME: I'm not sure this prop is suitable for this case, It might be better .
+  +optional?: boolean,
+|};
+
+function PricingCardAttributeItem({
+  className,
```

Comment:
It looks like className is unused, let's either added it to the call to `cn(...)` or remove it from props